tools/cloud_functions/gcs_event_based_ingest/tests/conftest.py

```diff
@@ -11,24 +11,20 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""unit tests for gcs_ocn_bq_ingest"""
+"""Integration tests for gcs_ocn_bq_ingest"""
 import json
 import os
-import sys
+import time
 import uuid
-from time import monotonic
 from typing import List
 
-import google.cloud.storage as storage
 import pytest
-from google.cloud import bigquery
-from google.cloud.exceptions import NotFound
+from google.cloud import bigquery, storage
```

Comment:
Imports should be on separate lines
https://google.github.io/styleguide/pyguide.html#313-imports-formatting

Reply:
Doh! we were just talking about this too!
Fixed here and in main.py in 348287f
I will look into if there is a pylint or isort plugin / configuration we can use to check this import style.